reuse/substitutions.yaml

```diff
@@ -0,0 +1,4 @@
+# Key/value substitutions to use within the Sphinx doc.
+{version_number: "0.1.0",
+ formatted_text: "*Multi-line* text\n that uses basic **markup**.",
+ site_link: "[Website link](https://example.com)"}
```

Comment:
Add a new-line to the end of this file, gets rid of the red marker on Github.
For reference, the new-line at the end of a file is recommended for proper formatting when you do something like `cat <file>` or append 2 files, etc.